introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I am Blauelf, trying to do a course on codeschool.com while they have their free weekend. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.